security-oauth2/src/main/java/io/micronaut/security/oauth2/client/clientcredentials/CacheableProcessor.java

```diff
@@ -52,22 +51,11 @@
     @NonNull
     private Queue<ElementSubscription<T>> subscriptions = new ConcurrentLinkedQueue<>();
 
-    @Nullable
-    private final Function<T, T> transformer;
-
     /**
      * Constructor.
      */
     CacheableProcessor() {
-        this.transformer = null;
-    }
 
-    /**
-     *
-     * @param transformer A transformation function to be applied when an element is received.
-     */
-    CacheableProcessor(@NonNull Function<T, T> transformer) {
```

Comment:
This seems a breaking change, although the class is `@Internal` and package-private. I guess it's ok

Reply:
Yes, I think it is fine because it is `@Internal `